main/index.html

```diff
@@ -138,12 +138,12 @@ <h1 class="title1">MetWork Framework</h2>
                 </div>
                 <!-- layer 2 -->
                 <div class="layer-1-2 wow slideInUp" data-wow-duration="2s" data-wow-delay=".1s">
-                  <h2 class="title2">An opensource framework for building Linux production systems in Meteorology, IoT or other domains.</h1>
+                  <h2 class="title2">Save time in building, running and monitoring your applications</h1>
```

Comment:
I agree with the drop of "Meteorology, IoT or other domains" but I prefer to keep the main definition because first of all "MetWork is an opensource framework".

```suggestion
                  <h2 class="title2">An open source framework for building Linux production systems</h1>
```

I will reuse "Save time in building, running and monitoring your applications" later

Reply:
I think it is not informative enough on MetWork features. We talk about open source and "battle tested" software bricks in the second and third subtitle. If you just say "come and see our nice open source framework", people will not understand what it is supposed to do.